### ISSUE TITLE
Lock/Disable Layer in SVG

### ISSUE THREAD

**Issue by webnologie** [methodofaction/Method-Draw#95]
Hello, is it possible to lock a layer in my own svg-file... example i have 3 objects and i wanna lock the last element of these.

thank you!

**Comment by RyAndrew** [comment]
Yes please! Locking background image position so you can't accidentally drag a background image would be awesome.
